src/ol/controls/Copyright.js

```diff
@@ -3,6 +3,31 @@ import CommonControl from '../../common/controls/Control';
 import mixin from '../../common/controls/mixins/Copyright';
 
 class Copyright extends mixin(CommonControl) {
+  defineProperties(opts) {
+    super.defineProperties(opts);
+    let { active } = opts;
+    Object.defineProperties(this, {
+      active: {
+        get: () => {
+          return active;
+        },
+        set: (newActiveVal) => {
+          active = newActiveVal;
+          if (newActiveVal) {
+            this.addCopyrightContainer(this.map.getTargetElement());
+            this.map.on('change:layers', this.onLayerChange.bind(this));
+            this.map.on('change:mobilityLayers', this.onLayerChange.bind(this));
+          } else {
+            this.removeCopyrightContainer();
+            this.map.un('change:layers', this.onLayerChange);
+            this.map.un('change:mobilityLayers', this.onLayerChange);
```

Comment:
in 'on' and 'un', it must be the same function

Reply:
I basically just moved the fonctions from activate/deactivate to the defineProperties.
Both uses 'this.onLayerChange'